aeon/dj_pipeline/acquisition.py

```diff
@@ -563,6 +563,47 @@ def make(self, key):
             )
 
 
+@schema
+class EnvironmentActiveConfig(dj.Imported):
```

Comment:
Would be nice if we can agree on whether to use `Config` or `Configuration` as a common suffix for all classes dealing with these files. I think I am fine either way.